core/arch/x86_code_test.c

```diff
@@ -134,7 +134,10 @@ unit_test_get_ymm_caller_saved()
     register __m256 ymm15 asm("ymm15");
 #        endif
 
-    for (int regno = 0; regno < proc_num_simd_registers(); ++regno) {
+    /* The function get_ymm_caller_saved is intended to be used for AVX (no AVX-512). It
+     * currently doesn't cover extended AVX-512 registers.
```

Comment:
"currently" implies it should and will be changed: so TODO i#?

Reply:
No, it should stay. I don't see a use case for ymm16-ymm31 right now? That's why I added the "currently". I'll remove the adverb.

Reply:
Done.